scripts/allocate-empty-primary

```diff
@@ -15,7 +15,7 @@ source env-prep
 
 node=${1:-}
 if [ -z "${node}" ] ; then
-  node=$(oc -n $LOGGING_NS get dc -l component=es -o jsonpath={.items[0].metadata.name})
+  node=$(oc -n $LOGGING_NS get dc -l component=$ES_COMPONENT -o jsonpath={.items[0].metadata.name})
```

Comment:
This depends on which version you are using 3.x vs 4.x.  It's different in both versions but constant to a particular version.  I would revert this.